docs/content/querying/filters.md

```diff
@@ -133,6 +147,9 @@ Search filters can be used to filter on partial string matches.
 |dimension|The dimension to perform the search over.|yes|
 |query|A JSON object for the type of search. See below for more information.|yes|
```

Comment:
Hmm actually a lot of other filters don't have these kinds of tables and it'd be nice to have the notice consistent…

Maybe keep the notice and add something here like,

`|extractionFn|[Extraction function](#filtering-with-extraction-functions) to apply to the dimension|no|`